pb/ctrl_pb/impl.go

```diff
@@ -18,6 +18,14 @@ package ctrl_pb
 
 import "github.com/openziti/fabric/controller/xt"
 
```

Comment:
I'm thinking it's only used in the router, so this should live somewhere else. ctrl_pb is only for code related to ctrl channel communications data. It should probably live in xgress/ since it's a way for xgress components to communicate back to the dialer.

Since it's wrapping an error, it should also implement `Unwrap`, see https://pkg.go.dev/errors